haas/ext/switches/brocade.py

```diff
@@ -277,8 +277,8 @@ def _construct_tag(name):
         """ Construct the xml tag by prepending the brocade tag prefix. """
         return '{urn:brocade.com:mgmt:brocade-interface}%s' % name
 
-    def _make_request(self, method, url, data=None):
+    def _make_request(self, method, url, data=None, logging=True):
```

Comment:
please use ``log`` instead of ``logging`` as it shadows the build in ``logging`` module.